src/podman.scss

```diff
@@ -30,6 +30,12 @@
     .pf-m-grid-md.pf-c-table [data-label] {
         justify-items: start;
     }
+
+    // Limit details width to 80 characters or 2/3 the width of the page,
+    // whichever is larger
+    .pf-c-description-list:not(:only-child) {
+        max-width: max(80ch,66ch);
```

Comment:
The comment does not match the code.

Reply:
...how does the comment not match the code? It explains exactly what it does.

Oh, I did make a typo though, it should be 80ch,66**vw**